lldb/unittests/Instruction/ARM64/TestAArch64Emulator.cpp

```diff
@@ -60,3 +169,18 @@ TEST_F(TestAArch64Emulator, TestOverflow) {
   ASSERT_EQ(pstate.V, 1ULL);
   ASSERT_EQ(pstate.C, 0ULL);
 }
+
+TEST_F(TestAArch64Emulator, TestAutoAdvancePC) {
+  Arch64EmulatorTester emu;
+  emu.memory_offset = 0x1234567800;
+  emu.gpr.pc = 0x1234567800;
```

Comment:
I see that this would be truncated if not for this fix, but please make it a full 64-bit value so that it is as obvious as it can be.